### PR TITLE
docs: update create review app.md

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Docs**

### Description

While triggering a review app noticed this new cloudflare feature that is SO useful IMO (screenshots on [slack thread](https://artsy.slack.com/archives/C2TQ4PT8R/p1678878896538269))

We are now able to add a descriptive comment about the review-app that we deploy and also tags that do not affect anything DNS related it is for our own reference.

Updated the docs to reflect that.

Happy to hear your thoughts on this :star-struck:

[slack thread](https://artsy.slack.com/archives/C2TQ4PT8R/p1678878896538269)

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/creating_review_app.md [hunk 1]
```diff
@@ -43,16 +43,18 @@ To access the review app, you must create a DNS name for it. The name must match
...
 1. Under `Proxy status` toggle to `Proxied`
+1. Under `Record Attributes > Comment` add the PR that created the review app, e.g. https://github.com/artsy/force/pull/123
+1. Under `Record Attributes > Tags` add the prefix of your team, e.g. `FX`
 1. Hit `Save`
 1. DNS will propagate and after a few minutes the review app will be available via `awesome-feature.artsy.net`
 
 ### Setting ENV variables on the review app
 
-To set an ENV variable, run: 
+To set an ENV variable, run:
```
Comment (gkartalis) [inline docs/creating_review_app.md:53]: linting  - ignore

File: docs/creating_review_app.md [hunk 2]
```diff
@@ -43,16 +43,18 @@ To access the review app, you must create a DNS name for it. The name must match
...
 
 ```sh
 hokusai review_app env set <my-review-app-name> SOME_ENV_VAR=true
-hokusai review_app refresh <my-review-app-name> 
+hokusai review_app refresh <my-review-app-name>
```
Comment (gkartalis) [inline docs/creating_review_app.md:57]: linting - ignore

**Review by dzucconi — APPROVED** [review]
Good idea

File: docs/creating_review_app.md [hunk 3]
```diff
@@ -43,16 +43,18 @@ To access the review app, you must create a DNS name for it. The name must match
 1. Under `Name` enter `awesome-feature`
 1. Under `Target` say `nginx-staging.artsy.net`
 1. Under `Proxy status` toggle to `Proxied`
+1. Under `Record Attributes > Comment` add the PR that created the review app, e.g. https://github.com/artsy/force/pull/123
+1. Under `Record Attributes > Tags` add the prefix of your team, e.g. `FX`
```
Comment (damassi) [inline docs/creating_review_app.md:47]: Nice 👍

**Pull request merged**